orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -291,4 +327,11 @@ public Status scan(String table, String startkey, int recordcount, Set<String> f
     }
     return Status.ERROR;
   }
+
+  /**
+   * Protected Access method to db variable for unit testing.
+   **/
+  protected ODatabaseDocumentTx getDB() {
```

Comment:
Should this be package protected so that the test can get it, buy a subclass wouldn't necessarily?